Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,10 @@
-﻿using System;
-
+﻿
 namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011723)